src/skins/vector/css/matrix-react-sdk/structures/_RoomView.scss

```diff
@@ -113,8 +118,6 @@ limitations under the License.
     max-width: 960px;
     margin: auto;
 
-    min-height: 100%;
```

Comment:
this worries me - isn't the min-height the only thing keeping the timeline contents pushed to the bottom of the page?

Reply:
I could have sworn I fixed this failure mode, but it seems to have been left out again. I'll try to fix it.

Reply:
Right, now I remember. I made the `.mx_messagePanelSpinner` specifically so that `mx_RoomView_messageListWrapper` could happily, separately use `min-height` without affecting the spinner.